setup.py

```diff
@@ -36,7 +36,7 @@ def read_file(file_name):
     version=read_file('hpolib/__version__.py').split()[-1].strip('\''),
     packages=setuptools.find_packages(exclude=['*.tests', '*.tests.*',
                                                'tests.*', 'tests'],),
-    python_requires='>=3.6, <3.8',
+    python_requires='>=3.6',
```

Comment:
have you tested it with a version > 3.8? I can remember, that there was a problem with some dependencies. But I am not sure if it is still the case. Maybe we should add another travis configuration with python3.8.X.

Reply:
I think Python 3.8's early versions have a bit of a rough history and were subsequently patched up. 3.8.3 was released in May and I've been using it for a while without any issues.